txt2array.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const { banksMap } = require('./clabe');
+
+let txt = fs.readFileSync('plazas.txt');
```

Comment:
It's not clear how these new files are supposed to be used.  What happens to the output?

Reply:
The .txt files are the data I got from STP, the js file only converts to json with the same form as in the clabe.js file. I think it would be better to have the catalog into the json file and require it into the clabe.js, what do you think about that?

Reply:
That's reasonable, but the steps need to be automated into the build process or at a minimum clearly documented.

It might be best to manually get the new city (plaza) codes into **clabe.js** now and then afterwards figure out how best to automate the process.

One option is to update this PR to just modify the `cities` array.  That'll be easy to merge and publish.

https://github.com/center-key/clabe-validator/blob/master/clabe.js#L209